env0/resource_environment.go

```diff
@@ -744,6 +751,7 @@ func deploy(d *schema.ResourceData, apiClient client.ApiClientInterface) diag.Di
 				Revision:             subEnvironment.Revision,
 				Workspace:            subEnvironment.Workspace,
 				ConfigurationChanges: configurationChanges,
+				UserRequiresApproval: !subEnvironment.ApprovePlanAutomatically,
```

Comment:
notice the '!'

Reply:
so what will happen when it is not there?

Reply:
the schema has a default value 'true'.
so it will be 'false'.